app/views/opendata/agents/nodes/member/show.html.erb

```diff
@@ -75,7 +75,7 @@ $ ->
           <% end %>
         </div>
         <nav>
-          <%= link_to :more, "#{@member_url}ideas/", class: "more", id: "idea-more" %>
+          <%= link_to t('ss.links.more'), "#{@member_url}ideas/", class: "more", id: "idea-more" %>
```

Comment:
__[brakeman]__
* Unsafe model attribute in link_to href [:ref](http://brakemanscanner.org/docs/warning_types/link_to_href)

<!-- Comment by [SideCI](https://sideci.com) -->